src/main/kotlin/racingcar/domain/CarName.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain
+
+class CarName(val name: String) {
+    init {
+        require(name.length <= VALID_NUMBER) { ERROR_MESSAGE }
+    }
+
+    override fun toString(): String {
+        return name
+    }
+
+    companion object {
+        const val ERROR_MESSAGE = "자동차 이름은 5자를 초과할 수 없습니다."
+        const val VALID_NUMBER = 5
```

Comment:
```suggestion
        private const val ERROR_MESSAGE = "자동차 이름은 5자를 초과할 수 없습니다."
        private const val VALID_NUMBER = 5
```

접근 제어를 private으로 해도 괜찮지 않을까요?